docs/installing-solidity.rst

```diff
@@ -60,17 +60,39 @@ Please refer to the solc-js repository for instructions.
 Docker
 ======
 
-We provide up to date docker builds for the compiler. The ``stable``
-repository contains released versions while the ``nightly``
-repository contains potentially unstable changes in the develop branch.
+Docker images of Solidity builds are available using the ``solc`` image from the ``ethereum`` organisation. Use the ``stable`` tag for the
+latest released version, and ``nightly`` for potentially unstable changes in the develop branch.
+The command below pulls the stable version of the `solc` image (if you don't have it already), and runs it in a new container,
+currently passing no input files or compiler options:
+
+.. code-block:: bash
+
+    docker run ethereum/solc:stable
+
+You can also specify specific release build versions in the tag, for example, for the 0.5.4 release.
+
+.. code-block:: bash
+
+    docker run ethereum/solc:0.5.4
+
+The Docker images contains the compiler executable, so you can pass all compiler arguments to it as an argument. For example.
```

Comment:
The fact that it contains the executable does not necessarily mean that that's what is run.